jerry-core/vm/vm.c

```diff
@@ -1733,6 +1733,10 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
 
             break;
           }
+          else if (ECMA_IS_VALUE_ERROR (result))
```

Comment:
The indentation is wrong (from line 1734 to 1742)

Reply:
I reordered the lines for better readability.